scripts/bundleContentScopeScripts.mjs

```diff
@@ -0,0 +1,13 @@
+import fs from 'fs/promises'
+const [targetPath, sourcePath, trackerLookupPath, configPath] = process.argv.slice(2)
+const utf8 = { encoding: 'utf-8' };
+
+(async () => {
+    const source = await fs.readFile(sourcePath, utf8)
```

Comment:
Nit: If we're waiting for each file to read in turn anyway, maybe just use the synchronous file APIs, which IIRC are quicker anyway + would mean we could skip wrapping in this async function.